docs/installation/debian.md

```diff
@@ -12,113 +13,121 @@ parent = "smn_linux"
 
 Docker is supported on the following versions of Debian:
 
+ - [*Debian testing stretch (64-bit)*](#debian-wheezy-stable-7-x-64-bit)
  - [*Debian 8.0 Jessie (64-bit)*](#debian-jessie-80-64-bit)
  - [*Debian 7.7 Wheezy (64-bit)*](#debian-wheezy-stable-7-x-64-bit)
 
-## Debian Jessie 8.0 (64-bit)
+ >**Note**: If you previously installed Docker using `apt`, make sure you update your apt` sources to the new `apt` repository.
 
-Debian 8 comes with a 3.16.0 Linux kernel, the `docker.io` package can be found in the `jessie-backports` repository. Reasoning behind this can be found <a href="https://lists.debian.org/debian-release/2015/03/msg00685.html" target="_blank">here</a>. Instructions how to enable the backports repository can be found <a href="http://backports.debian.org/Instructions/" target="_blank">here</a>.
+ ## Prerequisites
```

Comment:
extra space in front of the section marker. makes it not a section.
<img width="193" alt="screen shot 2015-10-02 at 18 32 43" src="https://cloud.githubusercontent.com/assets/13337345/10260851/01d419e6-6934-11e5-818a-d8c94ba59097.png">